plugins/org.jkiss.dbeaver.ext.oceanbase/OSGI-INF/l10n/bundle.properties

```diff
@@ -209,3 +209,369 @@ tree.users.node.name                 = \u7528\u6237
 tree.variable.node.name              = \u53D8\u91CF
 tree.view.node.name                  = \u89C6\u56FE
 tree.views.node.name                 = \u89C6\u56FE
+
+editor.package.declaration.name = \u58F0\u660E
+editor.package.declaration.description = \u58F0\u660E\u6E90
+editor.package.body.name = \u4E3B\u4F53
+editor.package.body.description = \u4E3B\u4F53\u6E90
+editor.scheduler.job.log.name = \u8FD0\u884C\u65E5\u5FD7
+editor.scheduler.job.log.description = Job \u8FD0\u884C\u8BE6\u7EC6\u65E5\u5FD7
+
+tree.schemas.node.name = \u6A21\u5F0F
+tree.schema.node.name = \u6A21\u5F0F
+tree.tables.node.name = \u8868
+tree.table.node.name = \u8868
+tree.columns.node.name = \u5217
```

Comment:
This and many parameters below are already defined above in the same file. There are a lot of duplicates rows.

Reply:
I removed several duplications. There is basically no repetition now.